bindings/src/Capgemini.PowerApps.SpecFlowBindings/Configuration/TestConfiguration.cs

```diff
@@ -107,9 +108,9 @@ public Uri GetTestUrl()
         /// <returns>The user configuration.</returns>
         public UserConfiguration GetUser(string userAlias, bool useCurrentUser = true)
         {
-            if (useCurrentUser && this.currentUser.Value != null)
+            if (useCurrentUser && currentUser != null && currentUser.Alias == userAlias)
```

Comment:
I think we might actually want `currentUser` to be a dictionary. If I call `GetUser("alias a")` followed by `GetUser("alias b")` and then `GetUser("alias a")` again all within the same thread I'll likely get two different `alias a` users. This is despite `currentUser` being `true`.

Reply:
Leave this one with me - just testing locally.

Reply:
Very good point - will let you sort it out